tests/v2/test_wakunode.nim

```diff
@@ -139,9 +139,9 @@ procSuite "WakuNode":
     var completionFut = newFuture[bool]()
 
     await node1.start()
-    node1.mountStore()
+    node1.mountStore("")
```

Comment:
Is there a reason this can't be default argument? Passing empty string kind of ugly